CMakeLists.txt

```
@@ -345,6 +349,7 @@ if(USE_PTHREADS)
 		if(HAVE_PTHREAD AND NOT APPLE AND NOT HAIKU)
 			set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -pthread")
 			set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -pthread")
+			set(PKGCFG_LDFLAGS_PRIV "${PKGCFG_LDFLAGS_PRIV} -lpthread")
```

Not 100% sure, but it's probably supposed to be this:
```suggestion
			set(PKGCFG_LDFLAGS_PRIV "${PKGCFG_LDFLAGS_PRIV} -pthread")
```

---

pkg-config `Requires.private` field is only used when linking against a static library (or more accurate, when you use the `--static` flag in your call to `pkg-config`). If pthreads isn't being used directly by the consuming app/library, then `-lpthread` should be fine to resolve missing symbols.

With dynamic libraries it's a non-issue, since the dynamic lib will already be linked against `libpthread`

---

I'm aware of the role of `Libs.private`, I was just worried that `-lpthread` wouldn't set all of the correct linker flags, since the objects being pulled in from the static lib haven't been linked prior.

---

Digging through my pkgconfig directory as-is, it seems that the libraries installed are pretty evenly divided on this one, so I'm not gonna press it.